init.php

```diff
@@ -115,8 +115,8 @@ public function hook_fetch_feed($feed_data, $fetch_url, $owner_uid, $feed, $last
 
 		$ch = curl_init($fetch_url);
 
-		curl_setopt($ch, CURLOPT_TIMEOUT, defined('FEED_FETCH_TIMEOUT') ? FEED_FETCH_TIMEOUT : 15);
-		curl_setopt($ch, CURLOPT_CONNECTTIMEOUT, defined('FEED_FETCH_CONNECT_TIMEOUT') ? FEED_FETCH_CONNECT_TIMEOUT : 5);
+		curl_setopt($ch, CURLOPT_TIMEOUT, Config::get(Config::FEED_FETCH_TIMEOUT));
```

Comment:
What is this `Config::`? Where it defined?

Reply:
This is the new way to access properties in ttrss, they're no longer global env variables since around 1 month ago.